resources/energyplus/ProposedEnergy+.idd

```diff
@@ -14155,6 +14155,7 @@ AirflowNetwork:MultiZone:SpecifiedFlowRate,
 
 AirflowNetwork:MultiZone:Component:DetailedOpening,
       \min-fields 16
+      \extensible:5
```

Comment:
do we need to add a max fields? Have you checked E+ source code (or tested manually) to check if it does accept more than 4?

Reply:
Our model cpp file hardcodes it... Whatever that's fine then (I'd prefer if everything was from IDD for more flexibility but it doesn't matter). https://github.com/NREL/OpenStudio/blob/72156a5eb35ef8f5f8274b3180136edfcb3fe575/src/model/AirflowNetworkDetailedOpening.cpp#L239-L245